nuclio/config.py

```diff
@@ -278,6 +316,7 @@ def merge(self, config):
     def apply(self, skipcmd=False):
         for k, v in self.env.items():
             environ[k] = v
+        # TODO: add secrets to environ ?
```

Comment:
you cannot, at this point because the secret value is dereference at runtime, when function has been deployed.